Dockerfile

```diff
@@ -11,5 +11,5 @@ RUN npm ci --production
 # Copy app to source directory
 COPY . .
 
-USER node
+USER 1000
```

Comment:
Maybe a comment to clarify it's the node user if one isn't aware :)

Reply:
Good point; added now :)